README.md

```diff
@@ -1,24 +1,30 @@
 # endpoints [![Build Status](https://secure.travis-ci.org/endpoints/endpoints.png)](http://travis-ci.org/endpoints/endpoints)
-> a hypermedia API framework for javascript.
 
 [![NPM](https://nodei.co/npm/endpoints.png)](https://nodei.co/npm/endpoints/)
 
-# What is it?
-A hypermedia API framework compliant with the [json api](http://jsonapi.org) specification, compatible with any http server that supports express or hapi style request handlers.
+Endpoints is a hypermedia API framework compliant with the [json api](http://jsonapi.org) specification, compatible with any http server that supports express or hapi style request handlers.
+
+## What is it?
+
+Endpoints gives you a powerful set of tools for defining and structuring an API. Endpoints conforms to the JSON API standard, a set of conventions for writing efficient JSON-based APIs, so that your data can be transferred as efficiently as possible. Endpoints lets you define your API's routes and data model in a way that integrates with your existing web framework (*e.g.* [express](http://expressjs.com/) or [hapi](http://hapijs.com/)) and ORM (*e.g.* [Bookshelf](http://bookshelfjs.org/)).
```

Comment:
Can you remove one instance of efficient in this paragraph? *writing ~~efficient~~ JSON-based APIs*